config.py

```diff
@@ -0,0 +1,13 @@
+import json
+
+
+def get_config():
```

Comment:
Pulled out some frequently repeated logic. Could still probably get more DRY but it's in a better place.

Reply:
Good call!